subsys/storage/flash_map/flash_map.c

```diff
@@ -38,6 +38,9 @@ static const struct driver_map_entry  flash_drivers_map[] = {
 #ifdef CONFIG_SPI_FLASH_W25QXXDV
 	{SPI_FLASH_0_ID, CONFIG_SPI_FLASH_W25QXXDV_DRV_NAME},
 #endif
+#ifdef CONFIG_SPI_NOR_DRV_NAME
```

Comment:
SPI_NOR_SPI_FREQ_0 is not a Kconfig variable, it is defined in the fixup file only, it needs to be changed to DT_ afaik.

Reply:
Addressed.